Pollo/Views/Cards/CardCell.swift

```diff
@@ -113,6 +113,7 @@ class CardCell: UICollectionViewCell {
         timerLabel.snp.makeConstraints { make in
             make.centerX.equalToSuperview()
             make.bottom.equalToSuperview().inset(timerLabelBottomPadding)
+            make.height.equalTo(timerLabelFontSize)
```

Comment:
The label height should equal 17.0 (as the area is on Zeplin), just to avoid any cropping of the text when you constrain to the exact font size.